skate-plugin/src/main/kotlin/com/slack/sgp/intellij/ui/FileChoosing.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij.ui
+
+import com.intellij.openapi.fileChooser.FileChooserDescriptor
+import com.intellij.openapi.vfs.VirtualFile
+
+/**
+ * * Provides utility function for creating file chooser descriptors to select specific types of
```

Comment:
nit: formatting's off here

```suggestion
 * Provides utility function for creating file chooser descriptors to select specific types of
```